File: Baekjoon/18406.cpp
#include <iostream>
using namespace std;

int main() {
    string n;
    cin >> n;

    int left = 0;
    int right = 0;

    for (int i = 0; i < n.length() / 2; i++) left += (n[i] - '0');
    for (int i = n.length() / 2; i < n.length(); i++) right += (n[i] - '0');

    if (left == right) cout << "LUCKY\n";
    else cout << "READY\n";
    return 0;
}

File: Baekjoon/18406.kt
fun main() {
    val n = readLine()!!
    var left = 0
    var right = 0

    for (i in 0 until n.length / 2) {
        left += n[i].toInt()
    }

    for (i in n.length / 2 until n.length) {
        right += n[i].toInt()
    }

    if (left == right) println("LUCKY")
    else println("READY")
}